ui/pages/onboarding-flow/metametrics/metametrics.js

```diff
@@ -32,15 +31,17 @@ import {
 
 import { MetaMetricsContext } from '../../../contexts/metametrics';
 import {
-  Box as BoxComponent,
```

Comment:
This isn't a pattern we should encourage. We should be promoting the active replacement of deprecated components when a file is touched.